app/javascript/src/components/Dashboard/Notes/List/Note/Body.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { Typography } from "@bigbinary/neetoui/v2";
```

Comment:
Please use aliases.

Reply:
The folder structure is quite odd here. SInce `Notes/List/Note` is ambiguous. 
Let's restructure it as follows - 
1. Create a folder `Notes/Card`. Inside of that, use `index.jsx` as the entry point and split the current `NoteHeader, NoteBody, and NoteFooter` components to a compound component as `Card.Header, Card.Body and Card.Footer`.